Made some design changes

DanCQ opened the pull request:
All bugs fixed on this one.
Made design changes. 
If you like them you can merge.
It's a lot of changes, so it's fine if you don't want to.
I just wanted you to take a look at the changes.

DanCQ committed:
Create README.md

DanCQ committed:
Update README.md

DanCQ committed:
Update index.html

DanCQ committed:
Update style.css

DanCQ committed:
Update script.js

DanCQ committed:
Update index.html

code clean up

DanCQ committed:
Update style.css

Fixed responsive design issues.
Added beach like background, and treasure chest design w/ hover opened chest with treasure inside.

DanCQ committed:
Update script.js

Once I know how I'll try to get the chest to move around randomly.

DanCQ committed:
Update style.css

responsive design fixes for small screens

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

More responsive design fixes

DanCQ committed:
Update index.html

DanCQ committed:
Update style.css

Responsive design fix

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

trying to fix window cutting off

AdityaChaudhary3 commented:
Yeah , 
Sure, your issue and pull request are justified.
I will check all these changes and surely merge your pull request.
But please give me some time. I am a little busy for 2-3 days.

Thanks for contributing.
Keep it up.

DanCQ committed:
Update style.css

Made the closed chest better looking.

DanCQ committed:
Update style.css

A better chest:hover animation. Seems to open upwards, instead of in the middle.

DanCQ committed:
Update style.css

Improved the overall appearance of the chests, and gave them a weathered look.

DanCQ committed:
Update style.css

Added color depth to the ocean.
Organized the linear-gradient properties for better readability.

DanCQ committed:
Update style.css

Added a wobble animation to everything, to mimic the ocean waves. Hope you don't get nauseous. ^__^
Tried to make the puzzle box look more like a ship haul.

DanCQ committed:
Update style.css

Made the puzzle box into the shape of a ship. 
May cause unintended  responsive design issues the I'll have to fix later.

DanCQ committed:
Update index.html

Added empty divs in order to use them to create the ship body.

DanCQ committed:
Update style.css

Responsive design fixes

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

Trying to find the exact design fix

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

The answer?

DanCQ committed:
Update style.css

I think with this last thing I solve all responsive design issues.

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

Final touch ups.

DanCQ committed:
Update style.css

Minor fixes

DanCQ committed:
Update style.css

Final touches to responsive design. I think there should be no more bugs.

DanCQ committed:
Update style.css

unnecessary padding property.

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

After testing this version looks good on all of my devices.

DanCQ committed:
Update style.css

DanCQ committed:
Update style.css

Added transitions to see how they turn out.

DanCQ committed:
Update README.md

DanCQ committed:
Add files via upload

DanCQ committed:
Update index.html

DanCQ committed:
Update style.css

Still needs responsive design.

DanCQ committed:
Update script.js

DanCQ committed:
Rename style.css to treasure.css

DanCQ committed:
Rename script.js to treasure.js

DanCQ committed:
Update index.html

DanCQ committed:
Update treasure.js

Added more comments

DanCQ committed:
Update treasure.css

Added responsive design to .lose
Alphabetized everything.

DanCQ committed:
Update treasure.css

DanCQ committed:
Update treasure.css

DanCQ committed:
Update index.html

DanCQ committed:
Update treasure.css

DanCQ committed:
Update README.md

DanCQ committed:
Update README.md

DanCQ committed:
Update treasure.css

DanCQ committed:
Update treasure.css

Checking what works best.

DanCQ committed:
Delete img directory

DanCQ committed:
Add files via upload

DanCQ committed:
Update treasure.js

Added sound effects

DanCQ committed:
Update treasure.css

DanCQ committed:
Update index.html

Fixes unnecessary margin corrections.

DanCQ committed:
Update treasure.css

DanCQ committed:
Add files via upload

Added ocean sounds.

DanCQ committed:
Update treasure.js

Ocean sound function added

DanCQ committed:
Delete ocean.mp3

DanCQ committed:
Add files via upload

DanCQ committed:
Update treasure.js

DanCQ committed:
Update treasure.css

Added an author contact section

DanCQ committed:
Update index.html

DanCQ committed:
Update treasure.css

DanCQ committed:
Update index.html

DanCQ committed:
Update treasure.css

DanCQ committed:
Update index.html

DanCQ committed:
Update treasure.css

DanCQ committed:
Update treasure.css

AdityaChaudhary3 reviewed (commented):
Thanks, DanCQ
You did fantastic work on this project.
It is especially adding the random function.  I also want to do it.
You did well with CSS.
In future, I am looking for work with you on many Projects.
hope, you enjoy.
And sorry for the delay.

DanCQ commented:
Thanks. 
I'm learning too, and building a game is a great way to get to try out what I've learned. 
It helps to memorize it.
I am following you, so maybe we'll be able to work together in future ideas.

Lastly, I added a guessing score keeping function too.
I've run out of ideas, so I think the project is finished for now, for me.
